catalog/ui/catalog-ui-search/src/main/webapp/component/input/geometry/input-geometry.view.js

```diff
@@ -22,8 +22,11 @@ var _ = require('underscore');
 function convertUserValueToWKT(val){
     val = val.split(' (').join('(').split(', ').join(',');
     val = val.split('MULTIPOINT').map(function(value, index){
-        if (index % 2 === 1 && value.indexOf('((') !== -1) {
-            return value.split('((').join('(').split('),(').join(',').split('))').join(')');
+        if (index % 2 === 1 && value.indexOf('((') === 0) {
```

Comment:
Yeah, that should work.  I'll update it to just check `value.indexOf('((') === 0`.